30-remote-site-setup.sh

```diff
@@ -84,21 +84,23 @@ grep '^OSG_GRID="/cvmfs/oasis.opensciencegrid.org/osg-software/osg-wn-client' \
      /var/lib/osg/job-environment*.conf > /dev/null 2>&1
 cvmfs_wn_client=$?
 
-override_opts=()
+bosco_cluster_opts=()
 if [[ -n $OVERRIDE_DIR ]]; then
     if [[ -d $OVERRIDE_DIR ]]; then
-        override_opts=(-o "$OVERRIDE_DIR")
+        bosco_cluster_opts=(-o "$OVERRIDE_DIR")
     else
         echo "WARNING: $OVERRIDE_DIR is not a directory. Skipping Bosco override."
     fi
 fi
 
+[[ -n "$REMOTE_BOSCO_DIR" ]] && bosco_cluster_opts+=(-b "$REMOTE_BOSCO_DIR")
```

Comment:
```suggestion
[[ $REMOTE_BOSCO_DIR ]] && bosco_cluster_opts+=(-b "$REMOTE_BOSCO_DIR")
```

Same thing here.  No functional difference, but you might say it's easier on the eyes  :sunglasses: 

(It is good that you quote it on the right hand side though, where it would otherwise be subject to word splitting and pathname expansion.)